scaletest/templates/scaletest-runner/scripts/report.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash
+set -euo pipefail
+
+[[ $VERBOSE == 1 ]] && set -x
+
+status=$1
+shift
+
+case "${status}" in
+started) ;;
+completed) ;;
+failed) ;;
+*)
+	echo "Unknown status: ${status}" >&2
+	exit 1
+	;;
+esac
+
+# shellcheck disable=SC2153 source=scaletest/templates/scaletest-runner/scripts/lib.sh
+. "${SCRIPTS_DIR}/lib.sh"
+
+# NOTE(mafredri): API returns HTML if we accidentally use `...//api` vs `.../api`.
```

Comment:
👀 that sounds like a bug?

Reply:
We could consider it one. Funnily enough, Grafana has the same issue (you get page not found). 😄

Reply:
Not a blocker for this though

Reply:
https://github.com/coder/coder/issues/9877